lectures/L10-slides.tex

```diff
@@ -220,8 +220,8 @@ \part{Structures}
 \begin{verbatim}
 struct Complex
 {
-    double real;
-    double imaginary;
+    public double real;
```

Comment:
This one should have gone the other way: the reference to the public keyword should be removed from the next slide.